Libplanet/Store/IStore.cs

```diff
@@ -198,14 +198,20 @@ AddressStateMap states
         /// </summary>
         /// <param name="chainId">The chain ID.</param>
         /// <param name="address">The <see cref="Address"/> to get state references.</param>
+        /// <param name="highestIndex">The highest index of state references to get.</param>
+        /// <param name="lowestIndex">The lowest index of state references to get.</param>
+        /// <param name="limit">The maximum number of state references to get.</param>
```

Comment:
The docs should define what if these parameters take `null`.

Reply:
I elaborated [the docs](https://github.com/planetarium/libplanet/pull/545/files#diff-57312b1f61d94402d4e1306a57839b87R201).

Reply:
Seems still the case of `null` is not explained.

Reply:
The docs were missing during the rebase. I restored.